app/src/main/java/com/batch/mcs/finalproject/LoginActivity.java

```diff
@@ -0,0 +1,13 @@
+package com.batch.mcs.finalproject;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+public class LoginActivity extends AppCompatActivity {
```

Comment:
You have the same activity in two different PRs.... mmmm let's make this PR to be raised from the other one instead if it is really dependable